### PR TITLE
Fix(front): notes relation picker

### LINKED ISSUE #2015: Bug: Multiselect opening on notes
## Bug Description

https://github.com/twentyhq/twenty/assets/13139771/31e40b5f-5823-416d-a46b-ab456329ecee

## Expected behaviour

Dropdown should open on only one place at a time

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Demo:

https://github.com/twentyhq/twenty/assets/140154534/f72812d8-74bb-4a32-b376-888687a50192

Fixes #2015

### CODE REVIEW

File: front/src/modules/activities/notes/components/NoteCard.tsx [hunk 1]
```diff
@@ -71,17 +76,24 @@ export const NoteCard = ({
     ?.content.map((x: any) => x.text)
     .join('\n');
 
+  const fieldContext = useMemo(
+    () => ({ recoilScopeId: note?.id ?? '' }),
+    [note?.id],
+  );
+
```
Comment (bosiraphael) [inline front/src/modules/activities/notes/components/NoteCard.tsx:83]: Is useMemo really needed here? @charlesBochet

Comment (gitstart-twenty) [reply front/src/modules/activities/notes/components/NoteCard.tsx:83]: The prop's value is an object, and every time the parent re-renders, it appears as if the value has changed even if it hasn't. Therefore, we use `useMemo`. As for its necessity, it's debatable.

File: front/src/modules/activities/notes/components/NoteCard.tsx [hunk 2]
```diff
@@ -71,17 +76,24 @@ export const NoteCard = ({
...
-        onClick={() => openActivityRightDrawer(note.id)}
-      >
-        <StyledNoteTitle>{note.title ?? 'Task Title'}</StyledNoteTitle>
-        <StyledCardContent>{body}</StyledCardContent>
-      </StyledCardDetailsContainer>
-      <StyledFooter>
-        <ActivityRelationEditableField activity={note} />
-      </StyledFooter>
-    </StyledCard>
+    <FieldContext.Provider value={fieldContext as GenericFieldContextType}>
```
[the remark this answers is not in the document]
Comment (charlesBochet) [reply front/src/modules/activities/notes/components/NoteCard.tsx:85]: however while this is not ideal, as we have not implemented the multi relation field type for now I agree this is the best way to do it

**Review by charlesBochet — APPROVED** [review]
Thanks for the fix :)

**Pull request merged**